flink-runtime/src/test/java/org/apache/flink/runtime/state/StateBackendTestBase.java

```diff
@@ -644,14 +629,14 @@ public void testBackendUsesRegisteredKryoDefaultSerializerUsingGetOrCreate() thr
                 }
             }
 
-            assertTrue("Didn't see the expected Kryo exception.", numExceptions > 0);
+            assertThat(numExceptions > 0).as("Didn't see the expected Kryo exception.").isTrue();
         } finally {
             IOUtils.closeQuietly(backend);
             backend.dispose();
         }
     }
 
-    @Test
+    @TestTemplate
     public void testBackendUsesRegisteredKryoSerializer() throws Exception {
```

Comment:
```suggestion
    void testBackendUsesRegisteredKryoSerializer() throws Exception {
```
could be package private